dependencies.yaml

```diff
@@ -0,0 +1,67 @@
+# see https://github.com/rapidsai/dependency-file-generator for details
```

Comment:
I'd keep one comment line and remove the rest of the section headers like `# dependency files to create`, `# channels to add`, etc.
```suggestion
# Dependency list for https://github.com/rapidsai/dependency-file-generator
```